whole-tale/ngx-dashboard
# Restyled login view/footer, removed footer elsewhere

## Problem
Footer takes up a lot of screen real estate, and does not very closely match the footer on wholetale.org

Fixes #112 

## Approach
* Adjust login view to previous include info from footer
* Adjust footer to more closely match wholetale.org, and to only display on the login view
* Add a line to the login view: `By accessing Whole Tale, you agree to our Terms of Use.` with a link to the terms
* Mention / link to affiliate universities
* Mention Jetstream / Stata license on login view (do these need links as well?)

<img width="1438" alt="Screen Shot 2020-12-10 at 1 06 44 PM" src="https://user-images.githubusercontent.com/1413653/101820450-3f158580-3aec-11eb-84f5-fa454c09840a.png">

## How to Test
1. Checkout this branch locally, rebuild the dashboard
2. Navigate to the Login view
    * You should see that the styling on this page more closely matches that of wholetale.org
    * You should see that the footer on this page more closely matches that of wholetale.org
    * You should see that the login view now requires users to implicitly agree to the Terms of Use
    * You should see that the login view now mentions the Grant Number, Jetstream resources, and the Stata license (do these need links?)
    * NOTE: I found it difficult to change the background color on only the login view - if the subtle blue background feels odd, I can revisit to try and color it white, as it is in the mockup
3. Click on "Access WholeTale" and login to the WholeTale platform
    * You should see that the footer has been removed from all other views
4. Play around with other views to make sure that removing the footer didn't have any other weird side-effects elsewhere
